power_grid_model_c/power_grid_model/include/power_grid_model/main_model_impl.hpp

```diff
@@ -388,7 +388,6 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
             if (update_data.is_columnar(CT::name)) {
                 auto const buffer_span =
                     update_data.get_columnar_buffer_span<meta_data::update_getter_s, CT>(scenario_idx);
-                // [JGUO] ISSUE here: buffer_span .begin() and .end() are 0 away
```

Comment:
So what was the issue here?

Reply:
an `input` dataset was created, rather than an `update` dataset. As a result, `update_getter_s` returned an empty dataset